### PR TITLE
Make the Reservations page responsive

### PR DESCRIPTION
**Pull request by mrkamin**
Make the Reservations page responsive

### CODE REVIEW

File: src/components/reservation.css [hunk 1]
```diff
@@ -41,7 +44,23 @@
...
-    grid-template-columns: 1fr;
+  .reservation-page {
+    width: 100%;
+  }
+
+  .swiper-button-prev,
+  .swiper-button-next {
+    display: none;
+  }
+}
```
Comment (Muhammad0602) [inline src/components/reservation.css:55]: In the mobile version, you have all of your items in one row. However, that is not a good user experience. Everyone is accustomed to scrolling down when they have images like this. 
So, make sure you meet your users' needs by showing your items `vertically` not `horizontally`

File: src/components/reservation.css [hunk 2]
```diff
@@ -41,7 +44,23 @@
...
+
+@media (max-width: 768px) {
+  .reservation-page {
+    width: 100%;
+  }
+
+  .swiper-button-prev,
+  .swiper-button-next {
+    display: none;
```
Comment (Muhammad0602) [inline src/components/reservation.css:64]: [OPTIONAL] 
The image gets very big in width when the screen size gets closer to tablet size. 
image

I would suggest you don't increase the image width, but instead give some margin to the container, so it is in the middle.

**Review by Muhammad0602 — CHANGES_REQUESTED** [review]
Hello @mrkamin, 
Good job, so far. There are some little issues you should work on, but you have done an amazing job.

Kindly check the comments below, for more information.

File: src/components/reservation.css (hunk 1 above)
Comment (mrkamin) [reply src/components/reservation.css:55]: hi @Muhammad0602 ,
thanks for your feedback,
I am not sure but, in the design, as I got it is horizontally and with slider kindly check the link once and let me know if I am worng
mobil responsiv

https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign

Comment (Muhammad0602) [reply src/components/reservation.css:55]: This is the main page. But ok, you can keep it as it is. It looks nice.

File: src/components/reservation.css (hunk 2 above)
Comment (mrkamin) [reply src/components/reservation.css:64]: your suggestion applies to
photo fixt

**Pull request merged**